cmp/bb589773-ddb6-421c-b08e-f15bd92eb5c9/component.lp

```diff
@@ -0,0 +1,117 @@
+(librepcb_component bb589773-ddb6-421c-b08e-f15bd92eb5c9
+ (name "TCA9548A I2C-MUX")
```

Comment:
I renamed the objects to `TCA9548A Breakout`

Reply:
Since the library does not act as a namespace for components, the user can't see that this component comes from the Adafruit library (which is important to know, as there might exist similar breakout boards from other companies). Thus the term "Adafruit" must be contained in the name.

> Maybe "Adafruit TCA9548A Breakout"?

:+1: